src/module/canvas/ghost-measured-template.ts

```diff
@@ -13,6 +13,13 @@ export class GhostTemplate extends MeasuredTemplatePF2e {
         const snapped = canvas.grid.getSnappedPosition(center.x, center.y, 2);
         this.document.x = snapped.x;
         this.document.y = snapped.y;
+        if (
+            this.moveTime === 0 &&
+            (canvas.grid.type === CONST.GRID_TYPES.HEXODDR || canvas.grid.type === CONST.GRID_TYPES.HEXEVENR)
+        ) {
+            this.document._source.direction += 30;
+            this.document.direction = this.document._source.direction;
```

Comment:
Use `this.document.updateSource` instead of assigning twice

Reply:
I am unaware of how this.document.updateSource(...) works as the original code did not use and there doesn't seem to be alot of examples. Would the following be correct?
```
            this.document.direction += 30;
            this.document.updateSource({ direction: this.document.direction });
```

Reply:
If you're adding to the value, you'd want:
```ts
this.document.updateSource({ direction: this.document.direction + 30 });
```
That will trigger data repreparation and set both.